hw4/hw4/ViewController.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ViewController.swift
+//  hw4
+//
+//  Created by Илья Мунгалов on 11.03.2022.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // 1) Создать массив из кортежей (Int, String).
+        
+        let myArray = [(2, "abcde"), (3, "abc"), (4, "a")]
+        var newArray = [Int]()
+        var filterNewArray = [(Int, String)]()
+
+        
+        let range = 0..<myArray.count
+
+        // Пройти по элементам с помощью цикла for-in, создав Range от 0 до количества элементов массива, и возвести все Int в квадрат. (Лучше создать новый массив и результаты записать туда).
+        for elements in range {
```

Comment:
It's better to call a cycle variable `index` not `elements` because it would be an array element index, not an element itself